include/core/parameters_multiphysics.h

```diff
@@ -170,6 +171,55 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
+  /**
+   * @brief Parameters for algebraic reinitialization of the interface
+   * used with the VOF solver.
+   */
+  struct VOF_AlgebraicInterfaceReinitialization
+  {
+    /// Enables/Disables the algebraic interface reinitialization.
+    bool enable;
+    /// Enables/Disables pvtu format outputs of the algebraic interface
+    /// reinitialization steps of the last simulated time-step.
+    /// The files are stored in a folder named @p
+    /// algebraic-reinitialization-steps-output located inside the @p output @p
+    /// path folder specified in the @p simulation @p control subsection.
```

Comment:
It's a bit weird to have ///,  you can't use /* and */?

Reply:
I think it also works, I'll change it!